roles/openshift_node/tasks/config.yml

```diff
@@ -16,6 +16,16 @@
     sysctl_file: "/etc/sysctl.d/99-openshift.conf"
     reload: yes
 
+# The base OS RHEL with "Minimal" installation option is
+# enabled firewalld serivce by default, it denies unexpected 10250 port.
+# Reference: https://bugzilla.redhat.com/show_bug.cgi?id=1740439
+- name: Disable firewalld service
+  systemd:
+    name: "{{ item }}"
+    enabled: no
```

Comment:
Don't we also want `state: stopped` ?

Reply:
The all task is completed after host reboot, so it will be expected `stopped` state in result. In my experience, firewalld service does not affect during configuration tasks.

Reply:
ok, that makes sense